code/datums/outputs.dm

```diff
@@ -0,0 +1,106 @@
+//NOTE: When adding new sounds here, check to make sure they haven't been added already via CTRL + F.
+
+/datum/outputs
+	var/text = ""
+	var/list/sounds = 'sound/items/airhorn.ogg' //can be either a sound path or a WEIGHTED list, put multiple for random selection between sounds
+	var/list/vfx = list('icons/sound_icon.dmi',"circle", HUD_LAYER) //syntax: icon, icon_state, layer
```

Comment:
change the type of this var to var/mutable_appearance/vfx = list('icons/sound_icon.dmi',"circle", HUD_LAYER)
in /datum/outputs/New() do the vfx = new(vfx[1], vfx[2], vfx[3])

Reply:
you cant do var/mutable_appearance/vfx = list('icons/sound_icon.dmi',"circle", HUD_LAYER) in define only set to null

Reply:
You can define any type var as init list in the type def unless the cast var type is mutual_appearance?
Sounds like the biggest bullshit in the universe to me, but given its byond i wouldnt say its impossible

Reply:
>/mob/living/carbon/test
var/mutable_appearance/mt = list("1","2","3")
	
>/mob/living/carbon/test/New()
	mt = new(mt[1], mt[2], mt[3])

This compiles

Reply:
Also in the worst case scenatio you can do this
>/mob/living/carbon/test
var/list/mt = list("1","2","3")

>/mob/living/carbon/test/New()
var/mutable_appearance/x = new(mt[1], mt[2], mt[3])
mt = x